src/agent/v8/debugapi.ts

```diff
@@ -41,12 +41,9 @@ interface DebugApiConstructor {
 }
 
 let debugApiConstructor: DebugApiConstructor;
-const nodeVersion = /v(\d+\.\d+\.\d+)/.exec(process.version);
 
-if (!nodeVersion || nodeVersion.length < 2) {
-  const dummyapi = require('./dummy-debugapi');
-  debugApiConstructor = dummyapi.DummyDebugApi;
-} else if (semver.satisfies(nodeVersion[1], '>=8')) {
+if (semver.satisfies(process.version, '>=8') &&
```

Comment:
Can you remove `./dummy-debugapi` if it's no longer being used?

Reply:
Done.